Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Set default values for build arguments
+ARG DOCKERFILE_VERSION=1.0.0
+ARG JNLP_VERSION=4.3-4
+
+FROM jenkins/jnlp-slave:$JNLP_VERSION-alpine
+
+USER root
+
+# Install dependencies as root
+RUN apk add --no-cache python3-dev && \
+    apk add --no-cache --virtual .build-deps libffi-dev openssl-dev gcc libc-dev make && \
+    pip3 install --no-cache-dir --upgrade pip && \
+    pip3 install --no-cache-dir docker-compose && \
+    apk del .build-deps
+
+# Note: Latest version of kubectl may be found at:
+# https://github.com/kubernetes/kubernetes/releases
+ENV KUBE_LATEST_VERSION="v1.18.3"
```

Comment:
Could we remove the word `latest` and just use version?  First to be consistent with the Helm value, but also if we expose these through an argument instead of an environment variable, ie,` ARG KUBE_VERSION` we could allow builds to specify whichever version they like without us needing to change the image.

Reply:
Updated to ARG but the CLI tools would be baked into the image so they'd have to use what's there, I think